runtime/configuration/bref-ext.ini

```diff
@@ -0,0 +1,12 @@
+extension=/opt/php-modules/bcmath.so
+extension=/opt/php-modules/ctype.so
+extension=/opt/php-modules/dom.so
+extension=/opt/php-modules/exif.so
+extension=/opt/php-modules/fileinfo.so
+extension=/opt/php-modules/ftp.so
```

Comment:
:bulb: I doubt that everybody wants to load ftp extension. Would be nice to give the choice between a "batteries included" with all extensions loaded, and a "minimal" profile with for example only opcache for for example, so everybody's in charge then to load their required extensions then.

@mnapoli we already had this discussion in the past when I wanted to make bref as lightweight as possible and I'm still agree with you that the entry level user of Bref should have all batteries included, yet it should be possible to push forward, as loading fewer extentions worth considering it also reduce the attack surface of the λ.

Let's take only dom extension. For the record it's disabled by default on Ubuntu 20, especially because it relies on [libxml](https://www.php.net/manual/en/dom.requirements.php)... which is unfortunately known to have a very impressive [CVEs record track](https://www.cvedetails.com/vulnerability-list.php?vendor_id=1962&product_id=3311&version_id=&page=1&hasexp=0&opdos=0&opec=0&opov=0&opcsrf=0&opgpriv=0&opsqli=0&opxss=0&opdirt=0&opmemc=0&ophttprs=0&opbyp=0&opfileinc=0&opginf=0&cvssscoremin=0&cvssscoremax=0&year=0&month=0&cweid=0&order=4&trc=75&sha=690616413addb5ee367aa0cb65ba9cf5f99fba04) :boom: .

Reply:
In any case, removing extensions right now would be a breaking change, so I don't think it's worth addressing in this PR.

Reply:
For me it's ok to have every of these core extensions available in the layer FS, but not loaded by default with the "minimal" profile. Because as the layers limit is set to 5, we cannot put every ext into a dedicated layer otherwise every power user will reach such limit quickly.